python/api/vrfq/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/tradeparadigm/sdks.git@30455f784966f86d022d600af6438ecac13f3a7d#egg=opyn&subdirectory=opyn
+git+https://github.com/tradeparadigm/sdks.git@30455f784966f86d022d600af6438ecac13f3a7d#egg=ribbon&subdirectory=ribbon
+git+https://github.com/tradeparadigm/sdks.git@c31d0102e35a9407cc26437dcb64674a6ed430fe#egg=friktion&subdirectory=friktion
```

Comment:
Just a consideration, not necessarily a request for change:
I have some doubts that we will always keep these commit SHA updated
Have them  pinned will ensure us that the example will continue to work even in case of non back-compatible changes to the HEAD but on the other side we could have pretty outdated examples

Reply:
Good point. I think it is better to install from HEAD as we are doing a lot of changes to the sdk, so it would get outdated soon.

Reply:
Changed the requirements to install from main but it seems that the signature is broken there. Would you have some context about it?
![Screen Shot 2022-08-04 at 11 24 42](https://user-images.githubusercontent.com/4041238/182871995-9c171271-1125-4a52-9149-6a7899e7e6e5.png)

Reply:
Somehow missed a parameter conversion during the refactor :| Fixed via https://github.com/tradeparadigm/sdks/pull/51

Reply:
Validated locally and it is fixed. Thanks 🚀